### PR TITLE
Add DOI, arxiv, dblp links to some papers.

### PR DESCRIPTION
**Pull request by andrejbauer**
Would this be a good way to addres issue #8 ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,8 +56,10 @@ Note: end a line with two spaces to force a line break.
 * **Do be do be do** (Draft, 2014)  
   by Sam Lindley and Conor McBride
 
-* **Inferring algebraic effects** ([LMCS 2014](http://lmcs-online.org/ojs/viewarticle.php?id=1469))  
-  ([paper](http://arxiv.org/pdf/1312.2334.pdf))  
+* **Inferring algebraic effects** (LMCS 2014)
```
Comment (yallop) [inline README.md]: Could you please add two blank spaces after the title here?

**Comment by yallop** [comment]
Yes, I think this is a good approach.

**Comment by andrejbauer** [comment]
I think the formatting now is better. There are two DOI links that are actually correct but the script fails on them. Any ideas?

**Comment by yallop** [comment]
The DOI server was refusing to respond to HTTP HEAD requests.  I've [updated the script](https://github.com/yallop/check-markdown-urls/commit/d4efda96655c588a244617dc880b1bce0e22004c) to fall back to GET if the HEAD request fails.

Thanks for the contribution!

**Pull request merged**